sdk/schemaregistry/schema-registry-json/review/schema-registry-json.api.md

```diff
@@ -14,7 +14,7 @@ export interface DeserializeOptions {
 // @public
 export class JsonSchemaSerializer<MessageT = MessageContent> {
     constructor(client: SchemaRegistry, options?: JsonSchemaSerializerOptions<MessageT>);
-    deserialize(message: MessageT, options?: DeserializeOptions): Promise<unknown>;
+    deserialize<T>(message: MessageT, options?: DeserializeOptions): Promise<T>;
```

Comment:
If you want to preserve the default when you don't pass T to be unknown you can do `deserialize<T = unknown>` that way you are not forced to give a template param to deserialize

Reply:
`T` will default `unknown` if not specified.

Reply:
Oh yeah! Thanks for pointing it our @deyaaeldeen